src/lang/nl.json

```diff
@@ -45,6 +45,7 @@
           "absor": "U heeft absorptie niet correct toegepast.",
           "absor2": "U heeft absorptie niet correct toegepast: let op welke subformule u overhoudt.",
           "andcompl": {
+            "#": "U heeft een complement regel niet correct toegepast.",
             "inv": "U heeft een complement regel niet correct toegepast."
```

Comment:
Ik heb hier ook niet `.inv` keys gemaakt. Ik weet niet of die ook gebruikt worden in de backend. Met demorgan3 rn demorgan4 zijn ze inderdaad anders met conjunctie/disjunctie maar ik weet niet wat ik met de anderen moet doen.